ee/dcos-versions/1.11.0/dcos-bootstrap/templates/upgrade/run.sh

```diff
@@ -103,7 +103,7 @@ cp /tmp/fault-domain-detect genconf/. &> /dev/null; if [[ $? -ne 0 ]]; then echo
 cp /tmp/license.txt genconf/. &> /dev/null; if [[ $? -ne 0 ]]; then echo "skipping absent /tmp/license.txt file"; else echo "copied file /tmp/license.txt to ~/genconf"; fi
 OVERRIDE_PREVIOUS_DCOS_VERSION=${dcos_previous_version}
 # TODO(bernadinm) Terraform Bug: 9488. Templates will not accept list, but only strings. Used for PREVIOUS_DCOS_VERSION generation.
-PREVIOUS_DCOS_VERSION=$(curl -ksL ${element(compact(split("\n - ","${dcos_master_list}")),"${dcos_previous_version_master_index}")}/dcos-metadata/dcos-version.json | grep version | cut -d ":" -f2 | sed 's/ //g' | sed 's/,$//' | sed 's/\"//g')
+PREVIOUS_DCOS_VERSION=$(curl -ksL ${element(concat(compact(split("\n - ","${dcos_master_list}")), list("")),"${dcos_previous_version_master_index}")}/dcos-metadata/dcos-version.json | grep version | cut -d ":" -f2 | sed 's/ //g;s/,$//;s/"//g')
```

Comment:
how can we end up in a empty master list. I'm a bit confused about this change

Reply:
This is required since the remote bootstrap will always need to know the master private ips: https://github.com/dcos-terraform/terraform-aws-remote-agents/commit/fdc033b1d03e3721be0f29cd4ad2bb3c41b576ed#diff-0126f02549b2c056c62f7dd88dc423f8R136

This will give the option to simply not require the master private ips in the remote region.

Reply:
yeah, its evaluated for the local region and the ip is there, but for the remote region it gets evaluated with no IPs, but at least it doesn't just fail which this helps fixes the failure problem.